### ISSUE TITLE
Weird markup parsing

### ISSUE THREAD

**Issue by ggreif** [melioratus/org-mode#7]
The following snippet
```
#+BEGIN_NOTES
./SimuWinder +RTS -N5 -M30M -A10M -G2 -K2K -xc -s -p -hc -L40 -RTS --file recording-4packs.txt --replay
#+END_NOTES
```
will render very differently in `atom` and `emacs`.

**Comment by melioratus** [comment]
@ggreif 

This issue is fixed in version 0.3.12

Please download the update.

Thanks for your help!

**Comment by ggreif** [comment]
@melioratus Yep, this seems to be fixed, thanks!
